percona/controller/pgcluster/backup_test.go

```diff
@@ -0,0 +1,199 @@
+package pgcluster
+
+import (
+	"context"
+	"strconv"
+	"testing"
+
+	appsv1 "k8s.io/api/apps/v1"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	"github.com/percona/percona-postgresql-operator/internal/naming"
+	v2 "github.com/percona/percona-postgresql-operator/pkg/apis/pgv2.percona.com/v2"
+	"github.com/percona/percona-postgresql-operator/pkg/apis/postgres-operator.crunchydata.com/v1beta1"
+)
+
+func TestBackupOwnerReference(t *testing.T) {
```

Comment:
Just wondering why didn't you cover this with envtest, but rather with the fake client?

Reply:
It's necessary to have the ready status for pods, which is not possible when using envtest.

Reply:
Aha, I see. Thx